src/mgr/Mgr.cc

```diff
@@ -322,6 +325,29 @@ void Mgr::init()
     "Dump mgr status");
   ceph_assert(r == 0);
 
+  dout(4) << "Using sqlite3 version: " << sqlite3_libversion() << dendl;
```

Comment:
might want to define a macro `HAVE_LIBCEPHSQLITE` in `config-h.in.cmake` and guard following block with it?

Reply:
ACK, fixed.